__docs__/wonder-blocks-typography/typography.stories.tsx

```diff
@@ -73,34 +78,62 @@ export const ControlProps: StoryObj<typeof Title> = {
     },
 };
 
-export const TypographyElements: StoryObj<any> = () => (
-    <View>
-        <Title>Title</Title>
-        <HeadingLarge>HeadingLarge</HeadingLarge>
-        <HeadingMedium>HeadingMedium</HeadingMedium>
-        <HeadingSmall>HeadingSmall</HeadingSmall>
-        <HeadingXSmall>HeadingXSmall</HeadingXSmall>
-        <BodySerifBlock>BodySerifBlock</BodySerifBlock>
-        <BodySerif>BodySerif</BodySerif>
-        <BodyMonospace>BodyMonospace</BodyMonospace>
-        <Body>Body</Body>
-        <LabelLarge>LabelLarge</LabelLarge>
-        <LabelMedium>LabelMedium</LabelMedium>
-        <LabelSmall>LabelSmall</LabelSmall>
-        <LabelXSmall>LabelXSmall</LabelXSmall>
-        <Tagline>Tagline</Tagline>
-        <Caption>Caption</Caption>
-        <Footnote>Footnote</Footnote>
-    </View>
-);
+/**
+ These are all the available Thunderblocks typography elements with their names
+ written out in their respective styles. Wrapping them in `ThemeSwitcher` with
+ `theme ="thunderblocks"` will include the Plus Jakarta Sans typeface, otherwise
+ they will default to Lato.
+ */
+export const NewTypographyElements: StoryObj<any> = {
+    render: () => (
+        <View>
+            <Heading size="xxlarge">Heading</Heading>
+            <BodyText>BodyText</BodyText>
+        </View>
+    ),
+    globals: {theme: "thunderblocks"},
+    parameters: {
+        chromatic: {
+            // Disabling because the new typography components are covered
+            // in the Heading / BodyText stories
+            disableSnapshot: true,
+        },
+    },
+};
 
-TypographyElements.parameters = {
-    docs: {
-        description: {
-            story: `These are all the available typography elements
-            with their names written out in their respective styles.`,
+/**
+These are all the available classic typography elements with their names written
+out in their respective styles.
+ */
+export const ClassicTypographyElements: StoryObj<any> = {
+    parameters: {
+        chromatic: {
+            modes: {
+                default: allModes.themeDefault,
+                thunderblocks: allModes.themeThunderBlocks,
+            },
         },
     },
```

Comment:
same question related to a previous comment.... I couldn't see the snapshot changes in the recent Chromatic build.